### ISSUE TITLE
This core needs to be updated: major upstream fixes

### ISSUE THREAD

**Issue by vanfanel** [twinaphex/cannonball#21]
Hi there,

Could this core please be updated to latest sources from upstream in https://github.com/djyt/cannonball ?

There have been major changes, including graphics drawing inaccuracies being fixed.
Please, don't let this awesome core lag behind :)

**Comment by riggles1** [comment]
Yeah that custom music in newer versions would be amazing, the way RA has CRTSwitchres which is amazing for outrun, all these enhancements, but on a CRT at 60fps? Hot damn!
